pylint/config/option_manager_mixin.py

```diff
@@ -273,25 +274,7 @@ def read_config_file(self, config_file=None, verbose=None):
             parser = self.cfgfile_parser
```

Comment:
Might as well remove the `disable` in this PR.

Reply:
Yes sorry, I also rebased some commit from the other PR.